packages/assets-controllers/src/assetsUtil.test.ts

```diff
@@ -436,4 +439,79 @@ describe('assetsUtil', () => {
       expect(timestampsIncreasing).toBe(true);
     });
   });
+
+  describe('fetchAndMapExchangeRates', () => {
+    it('should return undefined exchange rates when chainId not supported', async () => {
+      // Arrange
```

Comment:
Nit: It is good to keep test phases in mind! Have you considered using empty lines to designate them instead of comments? See the example here: https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#-highlight-the-exercise-phase